README.md

```diff
@@ -0,0 +1,141 @@
+# [Arbitrum Token Bridge](https://bridge.arbitrum.io/)
+
+### The canonical token-bridge of [Arbitrum](https://arbitrum.io/)
+
+Are you a developer looking to explore the Arbitrum token bridge and its underlying technology? Then you've come to the right place! We'll walk you through the steps to run the Arbitrum token bridge website locally on your machine.
+
+<br />
+
+Arbitrum ❤️ Open-source
+
+Interested in contributing to this repo? We welcome your contribution.
+[Check out the contribution guidelines and instructions here](CONTRIBUTING.md).
+
+<br />
+
+---
+
+<br />
+
+## Prerequisites for running the code
+
+- Install [Node.js](https://nodejs.org/en/download/) for your platform
+- Install [VS Code](https://code.visualstudio.com/download) for your platform
+- Install [Yarn](https://classic.yarnpkg.com) - Package manager
```

Comment:
We should precise the version for Yarn and Node (or say that we can use `nvm use` to have the proper version)

Reply:
Good point. Added as a prerequisite and also in installation steps.